poc_iot_verifier/src/settings.rs

```diff
@@ -11,6 +11,9 @@ pub struct Settings {
     pub log: String,
     /// Cache location for generated verified reports
     pub cache: String,
+
+    #[serde(default = "default_denylist_url")]
+    pub denylist_url: String,
```

Comment:
shouldn't these be in the denylist settings module and then in the iot verifier's toml file under a `[denylist]` key? then the entry for iot verifier would just be `pub denylist: denylist: denylist::Settings,`